frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -500,6 +500,54 @@ export const getEducationType = (
     `Invalid educationType ${educationTypeId}`,
   );
 
+export const OperationalRequirementSearchDescription = defineMessages({
```

Comment:
I find the `SearchDescription` suffix a bit confusing here.  I know when Eric created the new search form he moved the old components into a deprecated folder.  I liked that if you did that here you wouldn't have to come up with new names them.